Yoda rewrite (#332)
ubvu/open-handbook: topics/yoda.qmd

categories: [Tools]
---

## What is it?
Yoda is an application for institutions that supports Research Data Management (RDM) throughout the entire research cycle: from the safe and easy storage and sharing of data during the research process, to the sharing of data within research collaborations and ultimately to research data archiving and publication.

{{< video https://www.youtube.com/embed/N67v2v62Qn0?si=MhjAv6lDHAyCgfUy >}}

> Yoda helps the researcher make their data "[FAIR](../topics/fair-principles.qmd)" by providing a solution that enables data discovery and sharing (i.e., findable, accessible). In addition, it facilitates the use of metadata, contributing to data interoperability and reusability. Yoda provides a platform for the implementation of standard workflows that can ensure metadata quality satisfying VU policy requirements for data archiving and publication. In addition, Yoda is built on [iRODS](https://irods.org) so it accommodates both researchers with data heavy requirements, as well as those seeking an accessible, user-friendly solution.

Reviewer: ```suggestion
Yoda helps the researcher make their data "[FAIR](../topics/fair-principles.qmd)" by providing a solution that enables data discovery and sharing (i.e., findable, accessible). In addition, it facilitates the use of metadata, contributing to data interoperability and reusability. Yoda provides a platform for the implementation of standard workflows that can ensure metadata quality satisfying [VU policy requirements](../topics/research-data-and-software-management-policy.qmd) for data archiving and publication. In addition, Yoda is built on [iRODS](https://irods.org) so it accommodates both researchers with data heavy requirements, as well as those seeking an accessible, user-friendly solution.
```

## What can it be used for?

The particular strength of Yoda lies in the fact that it is an all-in-one solution for managing your data during and after your project. It makes it easy to share and organize your data and add descripitve metadata at any moment. If (part of) your datasets needs to be published during or after your project this can be done by a few simple steps within Yoda itself, there is no need to upload your data to another platform. 

### Data storage
Yoda is a Cloud storage solution that can be used for small to very large datasets. The underlying [iRODS](https://irods.org) software ensures data integrity. Data in Yoda is backed up daily. 

### Data sharing
Once a new Yoda group has been created you can invite collaborators yourself. Researchers from national and international research institutes can login with their institutional account via [SURF Research Access Management (SRAM)](https://www.surf.nl/en/services/surf-research-access-management). Collaborators from non-research institutes can create a free EduID NL account. 

### Sensitive data

> The use of MFA with SRAM and hosting at SURF ensure, among other things, Yoda is suitable for the storage and sharing of Highly Sensitive data. Please make sure to contact the 🔒 [Privacy Champion](https://vu.nl/en/employee/privacy-and-information-security/privacy-champions-information) of your faculty to check if further measures are needed.

Reviewer 2: ```suggestion
The use of Multi-Factor Authentication (MFA) with SRAM and hosting at SURF ensure, among other things, Yoda is suitable for the storage and sharing of Highly Sensitive data. Please make sure to contact the 🔒 [Privacy Champion](https://vu.nl/en/employee/privacy-and-information-security/privacy-champions-information) of your faculty to check if further measures are needed.
```

It wasn't defined before.

> The use of MFA with SRAM and hosting at SURF ensure, among other things, Yoda is suitable for the storage and sharing of Highly Sensitive data. Please make sure to contact the 🔒 [Privacy Champion](https://vu.nl/en/employee/privacy-and-information-security/privacy-champions-information) of your faculty to check if further measures are needed.

Reviewer: ```suggestion
The use of MFA with SRAM and hosting at SURF ensure, among other things, Yoda is suitable for the storage and sharing of data that score High on confidentiality in a data classification (see the [Policy Classification of Research Data](https://rdm.vu.nl/guides/policies-supporting-vision-open-science.html#policy-classification-of-research-data) and the [Research Data Classification Tool](https://vu.nl/en/research/dataclassification)). Please make sure to contact the [RDM Support Desk](rdm@vu.nl) to check if further measures are needed.
```

### Metadata
At any moment you can add descriptive metadata to a folder in Yoda via a web form. This means you can easily organize your data in such a way that it is not only ready for archiving and publishing but also findable for your collaborators. 

### Archiving

> At any moment you can submit a data folder with metadata for Archiving in the Yoda **Vault**. Data in the Vault is read-only and cannot be deleted. To help make sure the dataset is suitable for archiving a Yoda **datamanager** (usually a data manager from you department or a faculty data steward) will review your submission before final approval. Data in the Vault is directly accessible for your collaborators. Access for the Yoda datamanager role ensures the data remains accessible on the long term.

Reviewer: ```suggestion
At any moment you can submit a data folder with metadata for Archiving in the Yoda **Vault**. Data in the Vault is read-only and cannot be deleted. To help make sure the dataset is suitable for archiving a Yoda **datamanager** (usually a data manager from your department or a faculty data steward) will review your submission before final approval. Data in the Vault is directly accessible for your collaborators. Access for the Yoda datamanager role ensures the data remains accessible on the long term.
```

### Publishing metadata and data
From the Yoda Vault you can submit the dataset for **publication**. A [DOI](https://www.doi.org/the-identifier/what-is-a-doi/) will be generated and registered with [DataCite](https://datacite.org/) together with the metadata. DataCite ensures your dataset becomes findable on the internet and citable.

> The DOI link will lead to a **landing page** in Yoda showing the metadata. The dataset itself can be Open, allowing direct download, Restricted, meaning other researchers need to contact you for access, or Closed, signalling the dataset is properly archived but not available for reuse. 

Reviewer: ```suggestion
The DOI link will lead to a **landing page** in Yoda showing the metadata. The dataset itself can be Open, allowing direct download, Restricted, meaning other researchers need to follow a data request procedure, or Closed, signalling the dataset is properly archived but not available for reuse. 
```

### Automated workflows
Since Yoda is build on iRODS you could also build (automated) ingest and analysis workflows using iRODS Rules and Policies. Please [contact IT for Research](mailto:itvo.it@vu.nl) if you need assistance.

## How to request access
Yoda is available for every VU researcher. If you need space to store data for your research project, you can [request it via the linked form](https://yoda.vu.nl/site/getting-started/requesting-space.html).

Once the project space has been created you can start to invite your collaborators yourself. Note that VU students must 🔒 [connect a token to SURFsecureID](https://services.vu.nl/kb_view.do?sysparm_article=KB0011809) to be able to log in.

## Are there costs involved?
The costs of storing data in Yoda are detailed on the 🔒 [VU website](https://vu.nl/en/employee/research-data-support/research-archiving-storage-cost-model). 

## Getting Started

The VU Yoda website has practical information on the use of the Yoda for users [starting with Yoda](https://yoda.vu.nl/site/getting-started/index.html).

> More information can be found on the [SURF User Knowledge base](https://servicedesk.surf.nl/wiki/display/WIKI/Yoda+Hosting)

Reviewer: ```suggestion
More information can be found on the [SURF User Knowledge base](https://servicedesk.surf.nl/wiki/display/WIKI/Yoda+Hosting).
```
